workspaces/arborist/package.json

```diff
@@ -74,18 +74,12 @@
   "bin": {
     "arborist": "bin/index.js"
   },
-  "//": "sk test-env locale to catch locale-specific sorting",
```

Comment:
I hate having to lose this comment but can't think of a better way to annotate that.

Reply:
We use this in arborist and the CLI. I was thinking we could have template-oss set it in the future (and document the reasoning there) since there doesn't seem to be any downsides of having it.